projects/canopy-test-app/src/app/app.module.ts

```diff
@@ -27,7 +28,7 @@ import {
   LgHintModule,
   lgIconAdd,
   lgIconArrowDown,
-  lgIconClose,
+  lgIconClose, lgIconFilter,
```

Comment:
I'm surprised that Prettier or ESLint aren't moving this onto a new line automatically...

Reply:
I have updated the lint staged config - it wasn't running on the test app